programs/bpf/c/sdk/llvm/llvm-docker/bin/clang

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env bash -ex
```

Comment:
`#!/usr/bin/env bash -ex` doesn't work in all environments iirc  gotta use this instead:
```bash
#!/usr/bin/env bash
set -ex
```